src/config/Config.coffee

```diff
@@ -950,7 +950,19 @@ Config =
     'Open Gallery': [
       'g'
       'Opens the gallery.'
-    ]
+    ],
+    'Next Gallery Image': [
+      'Right',
+      'Go to the next image in gallery mode'
+    ],
+    'Previous Gallery Image': [
+      'Left',
+      'Go to the previous image in gallery mode'
+    ],
+    'Advance Gallery': [
```

Comment:
Didn't know what this means, so I have no problem changing the description to actually describe it.